int8 pin_id
int8 value
---
bool success